common/lib/dependabot/update_checkers/base.rb

```diff
@@ -141,6 +141,23 @@ def vulnerable?
         security_advisories.any? { |a| a.vulnerable?(version) }
       end
 
+      def ignore_conditions
+        @ignore_conditions.select do |ic|
```

Comment:
This seems eerily similar to https://github.com/dependabot/dependabot-core/blob/0352d543c2ed09ed24e42a72d6b0477b23a55dcf/common/lib/dependabot/config/update_config.rb#L21 , but has wildcard support!